grpc.go

```diff
@@ -718,44 +718,100 @@ func (a *agentGRPC) DestroySandbox(ctx context.Context, req *pb.DestroySandboxRe
 	return emptyResp, nil
 }
 
-func (a *agentGRPC) AddInterface(ctx context.Context, req *pb.AddInterfaceRequest) (*gpb.Empty, error) {
-	if err := a.sandbox.addInterface(nil, req.Interface); err != nil {
-		return emptyResp, err
-	}
+func (a *agentGRPC) AddInterfaces(ctx context.Context, req *pb.AddInterfacesRequest) (*pb.Interfaces, error) {
 
-	return emptyResp, nil
+	var resultingInterfaces pb.Interfaces
+
+	for _, ifc := range req.Interfaces {
+		if ifc == nil {
+			continue
+		}
+
+		err := a.sandbox.addInterface(nil, ifc)
+		if err != nil {
+			return &resultingInterfaces, err
+		}
+
+		resultingInterfaces.Interfaces = append(resultingInterfaces.Interfaces, ifc)
+	}
+	return &resultingInterfaces, nil
 }
 
-func (a *agentGRPC) RemoveInterface(ctx context.Context, req *pb.RemoveInterfaceRequest) (*gpb.Empty, error) {
-	if err := a.sandbox.removeInterface(nil, req.Name); err != nil {
-		return emptyResp, err
+func (a *agentGRPC) UpdateInterfaces(ctx context.Context, req *pb.UpdateInterfacesRequest) (*pb.Interfaces, error) {
+
+	var resultingInterfaces pb.Interfaces
+
+	for _, ifc := range req.Interfaces {
+		if ifc == nil {
+			continue
+		}
+
+		err := a.sandbox.updateInterface(nil, ifc)
+		if err != nil {
+			return &resultingInterfaces, err
+		}
```

Comment:
This is  ambiguous, consider this situation

UpdateInterfaces(A, B, C)

For B, you want to update "mac" and "mtu" and "add ip" for B but it failed, so in this implementation, 
`err := a.sandbox.updateInterface(nil, ifc)` return non-nil error, and `UpdateInterfaces` returns "{A}" and `err` as result.

But Interface B in this VM has gone into an unknown state, right? How do we know exactly the right state of all interfaces so?